Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    px = 1/(b-a) * jnp.ones(len(xs))
</pre><p>We should follow PEP8 naming convention, which says that variable name should be meaningful.</p><p>here px can be written as prob_x</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>